java/junit5/src/test/java/com/saucelabs/saucebindings/junit5/examples/LocalExample.java

```diff
@@ -0,0 +1,79 @@
+package com.saucelabs.saucebindings.junit5.examples;
+
+import com.saucelabs.saucebindings.SaucePlatform;
+import com.saucelabs.saucebindings.SauceSession;
+import com.saucelabs.saucebindings.junit5.SauceBindingsExtension;
+import com.saucelabs.saucebindings.options.SauceOptions;
+import java.time.Duration;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.junit.jupiter.api.extension.TestWatcher;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class LocalExample {
+  WebDriver driver;
+  SauceSession session;
+
+  public static SauceOptions createSauceOptions() {
+    ChromeOptions chromeOptions = new ChromeOptions();
+    chromeOptions.addArguments("--hide-scrollbars");
+
+    return SauceOptions.chrome(chromeOptions)
+        .setPlatformName(SaucePlatform.MAC_CATALINA)
+        .setIdleTimeout(Duration.ofSeconds(30))
+        .build();
+  }
+
+  public static Boolean useSauce() {
+    System.setProperty("sauce.enabled", "true");
+    return Boolean.getBoolean("sauce.enabled");
+  }
+
+  @RegisterExtension
+  static SauceBindingsExtension sauceExtension =
+      SauceBindingsExtension.builder()
+          .capabilities(createSauceOptions())
+          .enabled(useSauce())
```

Comment:
Do we have to call `.enabled()`? What happens if we don't? What if we didn't need to call this method and instead could enable/disable an extension by setting an environment variable. This variable will be read on our end and the appropriate behavior will be enabled. This approach is more user-friendly but could also lead to some confusion about how the Sauce extension is getting enabled/disabled

Reply:
This is to allow people to toggle local or remote execution in the same codebase. The idea is to pass in a boolean value to determine whether to use the Sauce extension or to use their local execution code. Maybe we rename it `useSauce()` instead of `enabled()`?

Reply:
Ok, it is enabled by default. It can be disabled by:
1. setting environment variable "SAUCE_DISABLED" to "true"
2. setting System Property "sauce.disabled" to "true"
3. Commenting out the registration lines 😂